string dataModule
string dataClass
string jsonData